Program/Seasons.py

```diff
@@ -0,0 +1,222 @@
+import time
+
+
+
+#This is the stored pictures, calling "global" so the entire program can use them.
+setMediaPath("F:\COMP 120 Tinkering Graphics-JH182233\Pictures") 
+picfour = makePicture("Tree-Four-Seasons.jpg") 
+picwinter = makePicture("Tree-Winter.jpg") 
+picspring = makePicture("Tree-Spring.jpg") 
+picsummer = makePicture("Tree-Summer.jpg")
+picautumn = makePicture("Tree-Autumn.jpg")  
+picM = makeEmptyPicture(513, 556, black)
+pic = makeEmptyPicture(1025, 1110, black)
+
+
+#The main function for the program
+def main():
+    copy()
+    request()
+    request2()
+    
+    
+
+def copy():
+#create an empty picture and populate it with others 
+    startX = 0
+    startY = 0 
+    repaint (pic)
+    time.sleep(1)  
+    copyInto(picspring, pic, startX, startY)
+    repaint (pic)
+    
+    startX = 512
```

Comment:
Defining arbitrary values as literals (and especially, repeating them) will result in maintainability issues as the program becomes larger. Define these as constants at the top of the function, or, make a comment to clarify their meaning, as appropriate.